lib/process-services/process-list/components/process-filters.component.spec.ts

```diff
@@ -196,4 +196,13 @@ describe('ProcessFiltersComponent', () => {
         expect(filterList.getCurrentFilter()).toBe(filter);
     });
 
+    it ('should select current process filter', () => {
```

Comment:
Could you please improve this unit test? 
We are just testing that a method has been called
I want to be sure that the behaviour is correct
so I have my filterList
I change the filterParam as input and 
the expected is currentFilter = the one I was looking for